Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/RemoteMongoClientIntTests.swift

```diff
@@ -1019,16 +1031,281 @@ class RemoteMongoClientIntTests: BaseStitchIntTestCocoaTouch {
         wait(for: [exp], timeout: 5.0)
     }
 
-    func testSync_find() {
-        // TODO: STITCH-2215
+    func testSync_Count() throws {
+        let coll = getTestColl()
+        let sync = coll.sync
+        sync.configure(conflictHandler: { _, _, rDoc in rDoc.fullDocument },
+                       changeEventDelegate: { _, _ in },
+                       errorListener: { _, _ in })
+
+        let joiner = CallbackJoiner()
+
+        sync.count(joiner.capture())
+        XCTAssertEqual(0, joiner.value())
+
+        let doc1 = ["hello": "world", "a": "b"] as Document
+        let doc2 = ["hello": "computer", "a": "b"] as Document
+        sync.insertMany(documents: [doc1, doc2], joiner.capture())
+        sync.count(joiner.capture())
+
+        XCTAssertEqual(2, joiner.value())
+        // TODO: STITCH-2262 Add delete case
```

Comment:
[nit] pull this TODO out into `testSync_Delete*()` function stubs, this is an odd place for the TODO

Reply:
As an integration test, I want to delete the documents and test the count has moved down to zero.